app/scripts/models/auth_brokers/fx-desktop-v1.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * V1 of the broker to communicate with Fx Desktop when signing in to Sync.
+ */
+
+define([
+  'lib/channels/fx-desktop-v1',
+  'lib/url',
+  'models/auth_brokers/fx-sync'
+], function (FxDesktopChannel, Url, FxSyncAuthenticationBroker) {
+  'use strict';
+
+  var FxDesktopV1AuthenticationBroker = FxSyncAuthenticationBroker.extend({
+    type: 'fx-desktop-v1',
+
+    _commands: {
```

Comment:
Long term, I'd like to move these tables into the channels created by the auth brokers. I would modify `send` to accept the command name, and the channel would morph the command name into something the receiver expects.